pkg/appregistry/input_test.go

```diff
@@ -0,0 +1,16 @@
+package appregistry
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDuplicatePackage(t *testing.T) {
+	input := []string{"Jaeger", "Jaeger"}
+
+	expected := []string{"Jaeger"}
+	actual := sanitizePackageList(input)
```

Comment:
nit: Since we don't have any unit test for this layer, can we add a test for `Parse` instead of the private function `sanitizePackageList`? This way we can verify `Parse` works as expected.

Reply:
Good idea - addressed in the latest commit.